ext/background/const.js

```diff
@@ -5,6 +5,15 @@ const BLOCK_LIST = { title: [/^chrome:\/\//, /^Range/] };
 const GUID_REGEX =
   '({){0,1}[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}(}){0,1}';
 
+const MERGE_BEHAVIOR = {
+  // Keep all existing fields, fill remaining fields with new values
+  MERGE_EXISTING: 'MERGE_EXISTING',
+  // Keep all new fields, fill remaining fields with existing values
+  MERGE_NEW: 'MERGE_NEW',
+  // Completely replace the object without checking the old copy
+  REPLACE: 'REPLACE',
+};
```

Comment:
I think that `MERGE_NEW` and `REPLACE` are identical given the behavior of the server (attachment fields that are not sent by the client are not over-ridden by the server).

```suggestion
const MERGE_BEHAVIOR = {
  // Keep all existing fields, fill remaining fields with new values from the client
  KEEP_EXISTING: 'KEEP_EXISTING',
  // Completely replace all fields from the client without checking the old copy
  REPLACE: 'REPLACE',
};
```

Reply:
Good point, I've updated the merge behavior to reflect this. There are now the 2 options you mentioned. Thanks for the reminder about how the server handles updates!